addons/finger/XEH_postInit.sqf

```diff
@@ -0,0 +1,23 @@
+#include "script_component.hpp"
+
+if (!hasInterface) exitWith {};
+
+GVAR(lastFPTime) = -1;
+GVAR(fingersHash) = HASH_CREATE;
+GVAR(pfeh_id) = -1;
+
+["SettingsInitialized", {
+    //If not enabled, dont't bother adding keybind or eventhandler
+    if (GVAR(maxRange) <= 0) exitWith {};
```

Comment:
I like how this is setup. However, I think something like `!GVAR(enabled)` instead of `GVAR(maxRange) <= 0` setting would probably be more user friendly, even if it's technically redundant.